tools/minibrowser/src/main/java/com/wpe/tools/minibrowser/Browser.kt

```diff
@@ -5,12 +5,10 @@ import android.content.res.Configuration
 import android.graphics.Color
 import android.os.Bundle
 import android.util.Log
-import android.view.KeyEvent
-import android.view.Menu
-import android.view.MenuItem
-import android.view.View
+import android.view.*
```

Comment:
In Java, star-imports are not considered as a good practice.
But I don't know what it is the status in Kotlin.

Reply:
there're are many new classes that need to be imported from android.view package and thus android studio auto formatted import to star